### ISSUE TITLE
Comment line on example

### ISSUE THREAD

**Issue by balarsen** [JSONheadedASCII/examples#1]
In simpleBGSM.dat I want to raise the discussion on the end of the header. 

 Epoch (UTC), Bx_GSM, By_GSM, Bz_GSM

Everything in the {} is fine. My issue is that header information seems like it should have leading # and the last line does not making it harder to skip. I would suggest that we amend our version of the standard to have all header information start with #.

**Comment by jbfaden** [comment]
I'm fine with that.  We should verify that these are easily read into IDL and Matlab.

**Comment by balarsen** [comment]
Agreed, we actually find that it is easier in language X with all the comments commented as you can give a skip '#' and then it works.

**Comment by drsteve** [comment]
I'm with Brian on this... The point of commenting the header is so we know that anything not commented is data, and anyone who doesn't care about the header just has a standard ASCII file. Reading the JSON part of the header also benefits from this as the bit that's valid JSON can be extracted fairly easily, then parsed, as long as the valid JSON comes first. Anything that comes after that can be trivially ignored.

I do have a further comment... the "old-style" header line
`Epoch (UTC), Bx_GSM, By_GSM, Bz_GSM`
has a space, which is appropriate since the data are comma-delimited. Most, if not all, of our JSON-headed ASCII files so far have been whitespace-delimited. I'm all for supporting different delimiters, but to be consistent with current codes and files we'd want to think about how to encode the delimiter in the header. My preference would be to keep whitespace delimiting as the default behaviour - i.e., in the reference implementation if no delimiter is encoded in metadata then it assumes whitespace.

I'm happy to add the matlab reader to my to-do list. If someone gets to it before me I won't be too upset though... My brief googling suggests that this is a rare case where a JSON-headed ASCII reader/writer might be easier to implement in IDL than MatLab. I'm probably going to be spending some time on MatLab Central...

**Comment by jbfaden** [comment]
Would you guys take some time to post more examples of the Rich ASCII files you are using?  I just happened across this group this morning, and it's been effective to get us thinking about this.  I think having a set of examples available to all in the wild will do nice things to lock this down.
